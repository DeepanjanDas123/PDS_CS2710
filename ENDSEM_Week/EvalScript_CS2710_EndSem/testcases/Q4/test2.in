GenerateLengthLimitedHuffman
qufhuobwk
3
Quit
