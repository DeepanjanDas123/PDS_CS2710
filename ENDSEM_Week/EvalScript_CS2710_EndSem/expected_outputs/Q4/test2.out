b 000
f 001
h 010
k 011
o 100
q 101
u 110
w 111
cost: 27
